views/score.erb

```diff
@@ -0,0 +1,38 @@
+<h1>Scrabble Score</h1>
+
+<form action="" method="post" accept-charset="utf-8">
+
+  <label for="word"> Scrabble Word </label>
+  <input type="text" name="scrabble[word]" value="" id="word">
+
+
+<!-- <label for="address_1">Address 1</label>
```

Comment:
Old or reused code should be removed when it is commented out and no longer needed.